truss-chains/truss_chains/remote_chainlet/utils.py

```diff
@@ -586,6 +587,23 @@ def __init__(self, websocket: "fastapi.WebSocket") -> None:
     async def close(self, code: int = 1000, reason: Optional[str] = None) -> None:
         await self._websocket.close(code=code, reason=reason)
 
+    async def receive(self) -> Union[str, bytes]:
+        try:
```

Comment:
This is pulled from lines [above](https://github.com/basetenlabs/truss/blob/7bd86adf6c0aab8898bbdb1bcaa4f828c723884b/truss-chains/truss_chains/remote_chainlet/utils.py?plain=1#L387-L390), since we need to expose WebsocketDisconnect to end users. 

This wrapper exists to try and hide FastAPI, but per a comment above we already knew we were leaking details via Exceptions. I'd rather not come up with a separate protocol for how users would have to think about catching websocket disconnects (especially since it would be different in Truss), so for now I just pass it through